db/donor_conditions.yml

```diff
@@ -1,9 +1,16 @@
 --- !omap
 New:
   :name_zh_tw: 全新
+  :visible_to_package: false
 Lightly Used:
   :name_zh_tw: 接近全新
+  :visible_to_package: false
 Heavily Used:
   :name_zh_tw: 較為殘舊
+  :visible_to_package: false
 Broken:
   :name_zh_tw: 需要維修
+  :visible_to_package: false
+Mixed:
+  :name_zh_tw: 混合的
+  :visible_to_package: true
```

Comment:
`visible_to_package` doesn't seem to describe what it does properly.

If you want it to only be available on stock and admin, maybe something like `internal` or `staff_only` makes more sense ?

cc @steveyken

Reply:
Yeah, would be better to call it `visible_to_donor` perhaps? (and flip the logic around)

Reply:
Make it a required non null field